lib/Bugsnag.js

```diff
@@ -98,38 +98,40 @@ export class Client {
     }
   }
 
-  setUser = (id, name, email) => {
+  setUser(id, name, email) {
     NativeClient.setUser({id, name, email});
   }
 
   /**
    * Clear custom user data and reset to the default device identifier
    */
-  clearUser = () => {
+  clearUser() {
     NativeClient.clearUser();
   }
 
   /**
    * Leaves a 'breadcrumb' log message. The most recent breadcrumbs
    * are attached to subsequent error reports.
    */
-  leaveBreadcrumb = (name, metadata) => {
-    if (name.constructor !== String) {
-      console.warn('Breadcrumb name must be a String');
+  leaveBreadcrumb(name, metadata) {
+    if (typeof name !== 'string') {
+      console.warn(`Breadcrumb name must be a string, got '${name}'. Discarding.`);
```

Comment:
I'd be happy to change this if I can make the message more helpful, but I think the variable itself is more useful than just the type of the variable. You can intuit the type from the content, but not vice versa.

One example of how this could be confusing is if `name` is `null`, then `typeof name` is `'object'`.